app/styles/components/_pdf-button.scss

```diff
@@ -1,19 +1,21 @@
+.pdf-button-container {
+    text-align: center;
 
-#pdf-button-container{
-    text-align:center;
-    .pdf-button{
-        border:0;
-        height:$button-large-height;
-        margin:$aside-margin;
-        padding:5px;
+    .pdf-button {
+        border: 0;
+        height: $button-large-height;
+        margin: $aside-margin;
+        padding: 5px;
         color: white;
         background-color: $gray-darker;
     }
-    .pdf-button:hover{
-        cursor:pointer;
+
+    .pdf-button:hover {
```

Comment:
Since you're using SASS, a better way to write this is:

```
.pdf-button {
    border: 0;
    height: $button-large-height;
    margin: $aside-margin;
    padding: 5px;
    color: white;
    background-color: $gray-darker;
    
    &:hover {
        cursor: pointer;
    }
}
```